src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -35,20 +35,26 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
 
   // fetch tasks when url params change
   useEffect(() => {
-    history.listen(location => {
+    return history.listen(async location => {
       if (networkStatus === NetworkStatus.ready && !error) {
-        fetchMore({
-          variables: getQueryVariablesFromUrlSearch(id, location.search),
-          updateQuery: (
-            prev: PatchTasksQuery,
-            { fetchMoreResult }: { fetchMoreResult: PatchTasksQuery }
-          ) => {
-            if (!fetchMoreResult) {
-              return prev;
+        // catch errors resulting from async requests
+        // that return after component unmounts
```

Comment:
it is standard for all async/await code to be wrapped in a try/catch; therefore these comments are not necessary